products/zoningtaxlots/bash/export.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+source bash/config.sh
```

Comment:
let's scrap this (and source ../../bash/utils.sh instead) - I see you had temporarily. It's not adding much and we reference `utils` directly in `build.sh` so let's be consistent here